earthfile2llb/interpreter.go

```diff
@@ -124,7 +125,7 @@ func (i *Interpreter) handleBlock(ctx context.Context, b spec.Block) error {
 }
 
 func (i *Interpreter) handleBlockParallel(ctx context.Context, b spec.Block, startIndex int) error {
-	if i.local {
+	if i.denyParallel {
```

Comment:
I wouldn't change this part, since we don't want to parallelize the commands within a single LOCALLY target. We want to parallelize multiple unrelated LOCALLY targets.